package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "asap",
+  "version": "1.0.0",
+  "description": "HueHueHue",
```

Comment:
Consider adding a meaningful description for the project instead of "HueHueHue". This will help others understand what the project is about. [important]